bim_webviewer/bim_web_viewer.md

```diff
@@ -1727,6 +1727,94 @@ Model
 
 ---
 
+### Set Object Color
+
+<p class="heading-link-container"><a class="heading-link" href="#set-object-color"></a></p>
+
+```js
+setObjectColor(palettes);
+```
+
+#### Description
+
+Changes object color in selcted, default, and xray mode.
```

Comment:
```suggestion
Changes object color in selected, default, and xray mode.
```

Reply:
Trying to figure out how this can be better described. I feel like we need to explain the different modes. Taking a stab for a few minutes.